lib/services/access_checker.go

```diff
@@ -774,10 +775,10 @@ func (a *accessChecker) EnumerateEntities(resource AccessCheckable, listFn roleE
 
 		result.wildcardDenied = result.wildcardDenied || wildcardDenied
 
-		if err := NewRoleSet(role).checkAccess(resource, a.info.Traits, AccessState{MFAVerified: true}); err == nil {
+		err := NewRoleSet(role).checkAccess(resource, a.info.Traits, AccessState{MFAVerified: true})
+		if err == nil {
```

Comment:
revert this to keep the blame off you? forgive me if i'm missing the point of this change